MekHQ/src/mekhq/campaign/Campaign.java

```diff
@@ -2239,29 +2239,38 @@ private PartInUse getPartInUse(Part p) {
         return (null != result.getPartToBuy()) ? result : null;
     }
 
-    private void updatePartInUseData(PartInUse piu, Part p) {
-        if ((p.getUnit() != null) || (p instanceof MissingPart)) {
+    private void updatePartInUseData(PartInUse piu, Part p, boolean ignoreMothballedUnits,
```

Comment:
I know this is an existing method, but let's replace `piu` and `p` with descriptive variable names.